src/components/gitPage/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useDispatch } from 'react-redux';
+import { updateUsername } from '../../actions';
+import './style.css';
+
+export const Gitpage = ({ handleSelect }) => {
+    const [ userName, setUsername ] = useState('')
+    const [userData, setUserData] = useState([])
+    const dispatch = useDispatch()
+
+    useEffect(() => {
+        const getUserDetails = async () => {
+            const my_token = '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'
```

Comment:
Is the token for the authenticated version of the api that has a higher request limit?